apps/frontend/src/views/Results.vue

```diff
@@ -162,10 +162,25 @@
         <v-row>
           <v-col xs-12>
             <v-card elevation="2">
-              <v-card-title>Results View Data</v-card-title>
+              <v-row>
+                <v-col>
+                  <v-card-title>Results View Data</v-card-title>
+                </v-col>
+                <v-col
+                  v-if="show_expand_all_button"
+                  cols="auto"
+                  class="text-right"
+                >
+                  <v-btn color="primary" text @click="expand_all"
+                    >Expand All</v-btn
+                  ></v-col
+                >
+              </v-row>
```

Comment:
It would be better if this lived with the Single Expand toggle (maybe as a toggle as well?)